### ISSUE TITLE
Issues with data

### ISSUE THREAD

**Issue by strogonoff** [geolexica/isotc211-glossary#49]
This concerns the `main` branch. I see there was a push yesterday, so it looks like it’s being actively worked on. In that case, here are some issues that break toolchains that work with that repository:

- UUIDs within files don’t match UUIDs in filenames:
  <img width="490" alt="Screenshot 2024-04-26 at 17 37 14" src="https://github.com/geolexica/isotc211-glossary/assets/292959/04d5b11b-773c-41c5-8e78-4c4d222eb7bd">
- UUIDs between concepts & localized concepts don’t match (concepts must refer to correct UUIDs of localized concepts).
  This may be because concepts are referencing the UUID in filename, whereas some software (e.g., RegistryKit, etc.) references the `id` from within the file
- Register items (at least localized concepts, I believe) are missing required top-level attribute `status`, which must be one of: `valid`, `invalid`, `superseded`, or `retired`. This means they are not recognized as register items by software that validates register contents more strictly

Minor issues—this does not break anything, but will cause unexpected diffs:

- The initial document separator `---` will be removed on data updates, because serializers that write to register YAML do not allow for multiple documents to be within one YAML file, and therefore omit the unnecessary separator

cc @**Hassan** @**Ronald Tse**

**Comment by ronaldtse** [comment]
@HassanAkbar if there are no `status` defined for those terms, just assume they are `valid`.

Ping @ReesePlews can we make that assumption? Thanks.

**Comment by ReesePlews** [comment]
hello @HassanAkbar i guess if the non-english term does not exist the status is "valid" but "unpublished". with the "valid" status coming from the base english term and the "unpublished" coming from no translation from the Danish group being submitted.   does this understanding make sense to everyone?  @ronaldtse do you think this is an acceptable understanding?

Hassan, please send me a link and i will have a check. we are still working from the staging site, correct?

Question: earlier when you are talking about the UUIDs and such, this does not mean there will be a change in the api (json, rdf, turtle) links, correct?

**Comment by strogonoff** [comment]
Just for the record, we need to make sure the status attribute value is supported by spec (register or glossarist). I have not heard of "unpublished", the closest I can think of is "invalid".On 13 May 2024, at 10:14 AM, Reese Plews ***@***.***> wrote:﻿
hello @HassanAkbar i guess if the non-english term does not exist the status is "valid" but "unpublished". with the "valid" status coming from the base english term and the "unpublished" coming from no translation from the Danish group being submitted.   does this understanding make sense to everyone?  @ronaldtse do you think this is an acceptable understanding?
Hassan, please send me a link and i will have a check. we are still working from the staging site, correct?
Question: earlier when you are talking about the UUIDs and such, this does not mean there will be a change in the api (json, rdf, turtle) links, correct?

**Comment by strogonoff** [comment]
I think it’s the register data model in question, so valid, invalid, superseded, retired… Unpublished items can exist as part of a proposal, but that’s about it I think.On 13 May 2024, at 7:33 PM, Anton Strogonoff ***@***.***> wrote:﻿Just for the record, we need to make sure the status attribute value is supported by spec (register or glossarist). I have not heard of "unpublished", the closest I can think of is "invalid".On 13 May 2024, at 10:14 AM, Reese Plews ***@***.***> wrote:﻿
hello @HassanAkbar i guess if the non-english term does not exist the status is "valid" but "unpublished". with the "valid" status coming from the base english term and the "unpublished" coming from no translation from the Danish group being submitted.   does this understanding make sense to everyone?  @ronaldtse do you think this is an acceptable understanding?
Hassan, please send me a link and i will have a check. we are still working from the staging site, correct?
Question: earlier when you are talking about the UUIDs and such, this does not mean there will be a change in the api (json, rdf, turtle) links, correct?

**Comment by HassanAkbar** [comment]
> Question: earlier when you are talking about the UUIDs and such, this does not mean there will be a change in the api (json, rdf, turtle) links, correct?

@ReesePlews yes that is correct, the api will not be changed just values of the ids will be updated.

> i guess if the non-english term does not exist the status is "valid" but "unpublished"

@ReesePlews The issue here is that the translation exists but it does not have a status attribute given in the excel file that is being used to generate the concepts. 

By the explanation provided by you and @strogonoff I believe that it should be `valid` in that case.
Is this correct?

cc: @ronaldtse

**Comment by ronaldtse** [comment]
Yes @HassanAkbar I believe this is the understanding. Thanks @ReesePlews @strogonoff !
